src/ImageReconstruction.jl

```diff
@@ -1,32 +1,22 @@
 module ImageReconstruction
 
+using Base.Threads
+using Interpolations
 using FFTW
 
 export radon, iradon
 
 """
-radon transform
+    radon(I::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
 
-https://en.wikipedia.org/wiki/Radon_transform
+Radon transform of a image `I` producing a sinogram from view angles `θ` in
+radians and detector sampling `t`.
 """
-function radon end
+function radon(I::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
+    P = zeros(eltype(I), length(t), length(θ))
+    nr, nc = size(I)
```

Comment:
```suggestion
    ax1, ax2 = axes(I)
```
so you support [arrays with arbitrary axes](https://julialang.org/blog/2017/04/offset-arrays/).